src/clojure_tictactoe/tictactoe.clj

```diff
@@ -1,86 +1,28 @@
 (ns clojure-tictactoe.tictactoe
```

Comment:
what's the planned responsibility of this namespace?

Reply:
See below.